### PR TITLE
Add billing feature documentation

### PR DESCRIPTION
**Pull request by RichardAkman**
Adds documentation for the platform billing feature. Also fixes typos in the authorization page.

### CODE REVIEW

File: docs/integrations/store/configu/authorization.mdx [hunk 1]
```diff
@@ -48,7 +48,7 @@ To assign roles to your [organization members](/integrations/store/configu/membe
 ## Creating a custom role
 
 Take control of your organization's access management with our flexible and dynamic custom roles feature.
-Every organization is unique, with specific requirements and workflows. That's why we empower you to create custom roles, allowing you to fine-tune access permissions according to your exact neeConstruction.
```
Comment (rannn505) [inline docs/integrations/store/configu/authorization.mdx:51]: is it related? or you just caught it along the way?
can you review all current configu platform documentation and let us know on mismatches?

File: docs/integrations/store/configu/billing.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Billing
+---
+
+The billing page allows organization owners to manage their organization's billing plan and track usage details.
+
+<Note>
```
Comment (rannn505) [inline docs/integrations/store/configu/billing.mdx:7]: Like!

File: docs/integrations/store/configu/billing.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  Configu Orchestrator is open-source and free forever. For the Configu Cloud, we offer a generous free "Starter" plan
+  and we expect that many developers and teams will fall within this plan's limits. In this page, you can track how much
```
Comment (rannn505) [inline docs/integrations/store/configu/billing.mdx:9]: currently, you cannot see "how much you use", adjust

**Review by rannn505 — APPROVED** [review]
@RichardAkman thanks, see minor comments

**Comment by rannn505** [comment]
@pelegpor please review!

File: docs/integrations/store/configu/authorization.mdx (hunk 1 above)
Comment (RichardAkman) [reply docs/integrations/store/configu/authorization.mdx:51]: Something i caught along the way, and of course, I'll try and hunt for more mismatches.

**Pull request merged**